docs/tutorials/get-started/blink-an-led.md

```diff
@@ -0,0 +1,416 @@
+
+### Install a Viam SDK
+
+Go ahead and install either the [Viam Python SDK](https://python.viam.dev/) or the [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk/robot/client#section-readme) on your local computer.
+Use which ever programming language you are most comfortable with.
+
+Refer to the appropriate SDK documentation for SDK installation instructions:
+
+- [Viam Python SDK](https://python.viam.dev/)
+- [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk/robot/client#section-readme)
+
+### Connect your robot to the Viam SDK
+
+The easiest way to get started writing an application with Viam is to navigate to your robot's page on the [Viam app](https://app.viam.com/robots) and select the **Code sample** tab.
+Select your programming language (**Python** or **Golang**), and copy the boilerplate code.
+
+{{% snippet "show-secret.md" %}}
+
+These code snippets import all the necessary libraries and set up a connection with the Viam app in the cloud.
+
+On your local computer, create a file called <file>blink.py</file> or <file>blink.go</file>.
+Paste the boilerplate code from the **Code sample** tab of the Viam app into the file in your code editor, and save the file.
+
+You can now run the code.
+Doing so will ensure that the Viam SDK is properly installed and that the `viam-server` instance on your robot is live.
+
+Run the code by typing the following command into the terminal, with the correct path to your file:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+python3 <INSERT PATH TO YOUR FILE>/blink.py
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+go <INSERT PATH TO YOUR FILE>/blink.go
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+If you successfully configured your robot and it is able to connect to the Viam app, you should see something like this printed to the terminal after running your program.
+What you see here is a list of the various {{< glossary_tooltip term_id="resource" text="resources" >}} that have been configured on your robot in the Viam app.
+
+<img src="../../img/blink-an-led/image4.png" alt="A screenshot from the Visual Studio Code command line that prints the output of print(robot.resource_names) when your Raspberry Pi has correctly connected and initialized with the Viam app. The output is an array of resources that have been pulled from the Viam app. Some of these are the Vision Service, Data Manager, and Board." width="500">
+
+### Write code to blink the LED
+
+In order to interact with the GPIO pins on our Raspberry Pi, you need to import the [board component](/components/board/) from the Viam SDK.
+
+The **Code sample** tab should have automatically added the board import for you, but it doesn't hurt to double-check.
```

Comment:
Yeah--I wasn't sure how to approach this because I want people to learn the need to import in case they write their own code from scratch, but should this section actually even be here? (other than the go time and fmt imports)?